handlers.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+)
+
+func HandlePokemonById(rw http.ResponseWriter, r *http.Request, ps []Pokemon) {
```

Comment:
You would like to name your handlers similar to the route they represent, like `GetPokemon`, since it's singular, it could be understood that your only getting one